Makefile

```diff
@@ -31,7 +32,17 @@ pkg-test:
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh && fetch_envtest_tools $(ENVTEST_ASSETS_DIR) && setup_envtest_env $(ENVTEST_ASSETS_DIR) && go test $(GO_FLAGS) -test.timeout=3m ./pkg/... -coverprofile cover.out
 
 integ-test:
-	go test $(GO_FLAGS) -test.timeout=5m ./tests/integration/... -coverprofile cover.out
+ifeq ($(TESTS), all)
+	go test $(GO_FLAGS) -test.timeout=30m ./tests/integration -v
```

Comment:
Missed this, -v is part of the GO_FLAGS, so it shouldn't be hardcoded.

Reply:
done